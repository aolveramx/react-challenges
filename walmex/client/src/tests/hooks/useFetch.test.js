import '@testing-library/jest-dom'
import { shallow } from 'enzyme'

describe('Tests in custom hook useFetch', () => {

  test('should render <useFetch />', () => {
    const wrapper = shallow(<useFetch />)
    expect(wrapper).toMatchSnapshot()
  });

})