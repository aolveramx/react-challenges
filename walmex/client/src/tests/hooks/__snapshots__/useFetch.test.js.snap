// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in custom hook useFetch should render <useFetch /> 1`] = `<useFetch />`;
